check/configuration/supported/trufflehog.md

```diff
@@ -18,12 +18,26 @@ layout:
 
 [**Trufflehog**](https://github.com/trufflesecurity/trufflehog) is a linter for Security.
 
-You can enable the Trufflehog linter with:
+
+trufflehog is composed of several linter commands.
+    
+`trufflehog` runs trufflehog normally.
+
+You can enable the `trufflehog` linter with:
 
 ```shell
 trunk check enable trufflehog
 ```
 
+`trufflehog-git` also runs trufflehog on the git history.
+
+You can enable the `trufflehog-git` linter with:
+
+```shell
+trunk check enable trufflehog-git
+```
+
```

Comment:
It may be worth also including a section here on the `trufflehog` action that we have, `trufflehog-pre-commit`, which prevents you from ever committing any secrets, but it can also be left out of this PR